Vision/Vision.bs

```diff
@@ -81,24 +74,23 @@ Boilerplate: omit conformance
 	in the standards we create.
 
 	Technology is not neutral;
-	new technologies enable new actions and new possibilities.
-	We are proud of the good the Web has enabled;
-	we must take the responsibility to use our values
+	new technologies enable new actions and new possibilities, and
+	we must take responsibility 
 	to address the actual impact of our work.
+	The W3C’s Technical Architecture Group's work 
+	to clearly define <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/">Ethical Web Principles</a> is a strong basis 
+	to improve the ethical integrity of the Web. 
 
-	Our vision is for a World Wide Web that is more inclusive,
-	and more respectful of its users:
-	a Web that supports truth over falsehood,
-	people over profits,
-	humanity over hate.
-
-	We will improve the ethical integrity of the Web.
-	As the Web continues to grow in importance
-	to include all humanity as its users,
-	it must increase its respect for those users
-	and the trust it earns from them.
+	The Web has had a tremendous impact on the world, 
+	and its impact will continue to grow in the future, 
+	as it expands reach, knowledge, education and services even more broadly.
```

Comment:
Oxford comma. Also “expands reach” is a bit of an awkward phrase, “reach” of what?

Reply:
Oxford comma resolved.